pegasus/sites.v3/code.org/public/educate/index.haml

```diff
@@ -1,35 +1 @@
----
-title: Educator Overview
-theme: responsive
-style_min: true
----
-
-%link{:href=>"/css/educate.css", :rel=>"stylesheet"}
-
-%h1 Teach Computer Science
-
-#educate-hero
-  %img.educate-banner{src: "/images/homepage/fill-945x300/girl10.jpg"}
-
-#educate-curriculum-grid
-  - curriculum_boxes = []
-  - curriculum_boxes << { title_s: 'Elementary school', description_t: 'Teachers say our five elementary courses are “totally awesome sauce” and fun for all students, even pre-readers.', url_s: '/educate/curriculum/elementary-school' }
-  - curriculum_boxes << { title_s: 'Middle school', description_t: 'Our new middle school course can be offered as a semester or year-long introduction to computer science for all students.', url_s: '/educate/curriculum/middle-school' }
-  - curriculum_boxes << { title_s: 'High school', description_t: 'We offer two year-long courses for high school: an intro course and an AP course. Both are designed to broaden participation in computer science.', url_s: '/educate/curriculum/high-school' }
-  - curriculum_boxes.each do |box|
-    = view :educate_box, item: box, color: "#59b9dc"
-
-%div{style: "clear:both"}
-
-%h1.resources-h1 Additional Resources
-
-#educate-resources-grid
-  - resource_boxes = []
-  - resource_boxes << { title_s: 'Hour of Code', description_t: 'Celebrated in December, but available year-round, the Hour of Code is the first step for anyone to learn that computer science is fun and accessible at all ages.', url_s: 'https://hourofcode.com' }
-  - resource_boxes << { title_s: 'Partner with us', description_t: 'Bring computer science to your country, state, region or school district.', url_s: '/educate/partner' }
-  - resource_boxes << { title_s: 'Beyond Code.org', description_t: 'There are many fantastic courses and organizations helping schools adopt computer science. See our list of recommended options.', url_s: '/educate/curriculum/3rd-party' }
-  - resource_boxes << { title_s: 'Inspire a student', description_t: 'Find a volunteer guest speaker, or use these videos, posters and how-to resources to inspire students to try computer science.', url_s: '/educate/resources/inspire' }
-  - resource_boxes << { title_s: 'Join the online community', description_t: 'Connect with other computer science teachers and stay up-to-date on what’s new with Code.org.', url_s: '/educate/community' }
-  - resource_boxes << { title_s: 'Tools and videos', description_t: 'These tools and videos can be used in any CS course to support learning.', url_s: '/educate/resources/videos' }
-  - resource_boxes.each do |box|
-    = view :educate_box, item: box, color: "#a69bc1"
+- redirect CDO.studio_url('/s/courses?view=teacher')
```

Comment:
Shouldn't this just be `/courses`? (same for others)

Reply:
No, /educate and /teach are the teacher's pages so they should go to `CDO.studio_url('/courses?view=teacher')` 

There is still a correction to remove the `/s` part though.